app/assets/javascripts/components/sidebar_footer.es6.jsx

```diff
@@ -0,0 +1,55 @@
+class SidebarFooter extends Component {
+
+  static get defaultState() {
+    return {};
+  }
+
+  get styles() {
+    return {
+      container: {
+        display: 'flex',
+        position: 'relative',
+        padding: '20px',
+        marginLeft: '14px',
+        alignItems: 'center',
+        justifyContent: 'center',
+      },
+      text: {
+        paddingRight: '5px',
+      },
+      img: {
```

Comment:
Let's change this to `image` instead of `img` because we try to avoid abbreviations when we can!